Src/AutoFixture.xUnit.net.UnitTest/DelegatingFixture.cs

```diff
@@ -101,5 +102,7 @@ public object Create(object request, ISpecimenContext context)
         }
 
         internal Func<object, ISpecimenContext, object> OnCreate { get; set; }
+
+        internal Action<ICustomization> OnCustomize { get; set; }

```

Comment:
Thanks @ploeh! That definitely make sense!

To be honest it took me a while to figure out how to test this scenario. The first thing that came into my mind was to inject an `IFixture` mock and check if the `Customize()` method is called, but the tests are mocking framework agnostic! I'm just wondering what is your motivation to design tests in such *fundamental* way?